charts/castai-db-optimizer/values.yaml

```diff
@@ -83,6 +83,8 @@ proxy:
   drainTimeSeconds: 60
   # -- Extra network debug logging.
   networkDebug: false
+  # -- Whether to use a Kubernetes TLS Secret (true) or the default self-signed certificate (false).
+  useCustomTlsCertificate: false
```

Comment:
maybe instead of "true/false" let's just make it `tlsCertificateSecret` , and mention in the comment it needs to have "tls.crt" and "tls.key" values.